packages/lib/models/Note.ts

```diff
@@ -202,13 +207,11 @@ export default class Note extends BaseItem {
 		const temp = [];
 		for (const p of pathsToTry) {
 			temp.push(p);
-			temp.push(markdownUtils.escapeLinkUrl(p));
+			temp.push(encodeURI(p));
 		}
 
 		pathsToTry = temp;
 
-		// this.logger().debug('replaceResourceExternalToInternalLinks', 'options:', options, 'pathsToTry:', pathsToTry);
```

Comment:
restore

Reply:
There's not much left to do before we can merge this PR. Any chance that can be fixed? I don't want this deleted because I often use it for debugging.

Reply:
Done